.github/workflows/pr-status-labels.yml

```diff
@@ -0,0 +1,27 @@
+name: [Test] Pull Request Label Status
+
+on: [pull_request, pull_request_review]
+
+jobs:
+  prhelper_job:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Label PR by status
+      id: runprhelper
+      uses: Matticusau/pr-helper@v1.2.6
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        enable-prmerge-automation: false
+        enable-prcomment-automation: false
+        enable-prlabel-automation: true
+        enable-prreviewer-frontmatter: false
+        enable-welcomemessage: false
+        welcome-message: "Thanks for opening an issue! Make sure you've followed CONTRIBUTING.md."
+        prmerge-requireallchecks: false
+        prmerge-requirereviewcount: 1
+        prmerge-method: 'merge'
```

Comment:
Do we need these settings if merge automation is disabled?

Reply:
We do. On my testing, for some reason the labeler part didn't work without having all the settings in the config, even if there were default settings by the action itself already set.